src/Post_Command.php

```diff
@@ -748,4 +748,31 @@ private function read_from_file_or_stdin( $arg ) {
 		}
 		return file_get_contents( $readfile );
 	}
+
+	/**
+	 * Get category id if given name/slug
```

Comment:
Maybe `Resolves post_category arg into an array of category ids.` or something better?